### ISSUE TITLE
Label_category_dict

### ISSUE THREAD

**Issue by pzaffino** [WAMAWAMA/WAMA_Modules#7]
Hi,
I'm interested in running a 3D VGG on a CT dataset (single channel).
I want to classify the images into 3 classes (0, 1, or 3).
I did not understand how the label_category_dict works.
How to set the number of classes?

Do you have a snippet of code for this scenario?

Thank you.
Best,

Paolo

**Comment by WAMAWAMA** [comment]
`label_category_dict ` is used to specify how many labels you want to predict, and how many categories each label has
e.g. for your single-label task of predicting 3 classes (0, 1, or 3), here is the demo code based on 3D VGG and single-channel input:

```python
import torch
import torch.nn as nn
from wama_modules.Encoder import VGGEncoder
from wama_modules.Head import ClassificationHead
from wama_modules.BaseModule import GlobalMaxPool


class Model(nn.Module):
    def __init__(self, in_channel, label_category_dict, dim=2):
        super().__init__()

        # encoder
        f_channel_list = [64, 128, 256, 512]
        self.encoder = VGGEncoder(
            in_channel,
            stage_output_channels=f_channel_list,
            blocks=[1, 2, 3, 4],
            downsample_ration=[0.5, 0.5, 0.5, 0.5],
            dim=dim)
        # cls head
        self.cls_head = ClassificationHead(label_category_dict, f_channel_list[-1])
        self.pooling = GlobalMaxPool()

    def forward(self, x):
        f = self.encoder(x)
        logits = self.cls_head(self.pooling(f[-1]))
        return logits


if __name__ == '__main__':
    input_channel = 1
    x = torch.ones([2, input_channel, 64, 64, 64])
    category_num = 3  # Because your label has 3 classes as you mentioned
    label_category_dict = dict(label_1=category_num)  # label_1 is your label name
    model = Model(in_channel=1, label_category_dict=label_category_dict, dim=3)
    logits = model(x)
    print('single-label predicted logits')
    _ = [print('logits of ', key, ':', logits[key].shape) for key in logits.keys()]
```
